grails-app/controllers/org/pih/warehouse/core/StockMovementItemsParamsCommand.groovy

```diff
@@ -11,6 +11,13 @@ class StockMovementItemsParamsCommand extends PaginationCommand implements Valid
 
     Boolean refreshPicklistItems = true
 
+    @Override
+    Integer getMax() {
```

Comment:
In my opinion we shouldn't override that. 
If it turned out, that the extension of `PaginationCommand` doesn't fit here, we should get rid of that extension instead of having to override methods in order to "cheat" the `PaginationCommand`.

What I would suggest, but I assume not everyone will like that idea, is to create a `PaginationSomething` interface, that you would implement here :smile: - then, I'd be fine with such override. 
Another option is - after creating an interface, we could specify two pagination commands - one, that would have the default values, and one, that wouldn't have them (I know that in ideal world we would have this behavior aligned).
P.S. I wouldn't have an idea how to name those, so I'm fine with just implementing the interface by yourself here.